src/shims-vue-others.d.ts

```
@@ -0,0 +1,9 @@
+import * as GlobFilters from '@/utils/filters';
+
+declare module '@vue/runtime-core' {
+  interface ComponentCustomProperties {
+    $filters: GlobFilters;
+  }
+}
+
+export { };
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)